client/src/pages/Home.js

```diff
@@ -102,18 +103,21 @@ const StyledP = styled.p`
     font-family: ${theme.typography.font.family.display}, sans-serif;
```

Comment:
I think the code is here but the font styles are different now on mobile

<img width="536" alt="Screen Shot 2020-05-10 at 11 53 43 PM" src="https://user-images.githubusercontent.com/42228386/81533363-eb0aae80-931a-11ea-8baf-e33e76c42b57.png">

Reply:
This seems to be fixed already